tests/drop_packets/test_drop_counters.py

```diff
@@ -38,20 +40,31 @@
 def enable_counters(duthosts, rand_one_dut_hostname):
     """ Fixture which enables RIF and L2 counters """
     duthost = duthosts[rand_one_dut_hostname]
-    cmd_list = ["intfstat -D", "counterpoll port enable", "counterpoll rif enable", "sonic-clear counters",
-                "sonic-clear rifcounters"]
-    cmd_get_cnt_status = "redis-cli -n 4 HGET \"FLEX_COUNTER_TABLE|{}\" \"FLEX_COUNTER_STATUS\""
 
-    previous_cnt_status = {item: duthost.command(cmd_get_cnt_status.format(item.upper()))["stdout"] for item in ["port", "rif"]}
+    previous_cnt_status = {}
+    # Separating comands based on whether they need to be done per namespace or globally.
+    cmd_list = ["intfstat -D", "sonic-clear counters"]
+    cmd_list_per_ns = ["counterpoll port enable", "counterpoll rif enable", "sonic-clear rifcounters"]
 
+    """ Fixture which enables RIF and L2 counters """
     for cmd in cmd_list:
         duthost.command(cmd)
```

Comment:
You can use shell_cmds() library to issue all commands at once.

Reply:
Updated with this change.